src/mugs.js

```diff
@@ -387,7 +387,13 @@ define([
                 if (lang !== defaultLang && disp === defaultDisp) {
                     disp += " [" + defaultLang + "]";
                 }
-                return $('<div>').text(disp).html();
+                if (this.options.richtext) {
+                    disp = disp.replace(/(<\/?p>)/ig,"");
+                    disp = disp.replace(/[\r\n|\n|\r]/ig," ");
+                    return $('<div>').html(disp).html();
```

Comment:
Currently reintroduces: https://github.com/dimagi/Vellum/pull/349 but only for those behind the feature flag. Need to come up with a whitelist of tags that aren't escaped for the future. (probably only `span`, `i`)